addons/spreadsheet/static/src/chart/plugins/odoo_chart_feature_plugin.js

```diff
@@ -103,6 +117,33 @@ export class OdooChartFeaturePlugin extends OdooUIPlugin {
         });
     }
 
+    getAvailableChartRegions(chartId) {
+        const definition = this.getters.getChartDefinition(chartId);
+        if (definition.type !== "odoo_geo" && definition.type !== "geo") {
+            return [];
+        }
+        if (!definition.region || definition.region === "world") {
+            this.worldCharts.add(chartId);
+        }
+        if (!this.worldCharts.has(chartId)) {
+            return [];
+        }
```

Comment:
If someone has a better idea to handle geo charts with world region that has been updated to another region...

Reply:
A store would also probably make more sense than a plugin for this feature also. Not sure it we can get the geoJsonService from a store tho ...